samples/shopify/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "contentful-ui-shopify",
+  "version": "1.0.2",
+  "description": "Contentful UI extension to select Shopify products",
+  "main": "index.js",
+  "engines": {
+    "node": ">=7.6"
+  },
+  "scripts": {
+    "create": "contentful-extension create --space-id ${SPACE_ID}",
```

Comment:
The contentful extension cli was deprecated. It would be great to have the contentful cli here. :)

https://github.com/contentful/extensions#commands